extensions/ch_vd_sire/delta/pre-all.py

```diff
@@ -0,0 +1,15 @@
+import os
+from pum.core.deltapy import DeltaPy
+
+
+class DropViews(DeltaPy):
```

Comment:
I like a lot the idea of setting drop/rewrite views like this, it's cleaner and more consistent.

Does PUM handle this already? 
Also, is it done in proper order (extensions should be run first on `pre` and and last on `post`)?

Reply:
We extended Pum so it can work with multiple delta directories, for the purpose of QWAT extensions effectively.

Currently, the execution order of `pre` and `post` is the same. We can easily change that in Pum, and have the execution of `post` being the reverse of `pre`. I'll have a closer look.

Reply:
yes, I think this is mandatory.